product/tutorials/tutorial_set_inventory.md

```diff
@@ -0,0 +1,105 @@
+# **Set Inventory**
+
+## Let's get started
+
+Changes to a product's inventory information may occur much more frequently than changes to its catalog information.
+
+You may decide to issue inventory updates instead of updating entire product.
+
+In such cases, the **```SetInventory```** method can be used.
+
+You can find detailed information about managing catalog information in the [Retail API documentation](https://cloud.google.com/retail/docs/inventory-updates#non-incremental-updates)
+
+
+**Time to complete**: 
+<walkthrough-tutorial-duration duration="3.0"></walkthrough-tutorial-duration>
+
+## Before you begin
+
+To run Python code samples from this tutorial, you need to set up your virtual environment.
+
+To do that, run the following commands in a terminal:
+
+```bash
+python3 -m venv tutorial-env
+```
+
+```bash
+source tutorial-env/bin/activate
+```
+
+Next, install Google packages:
+
+```bash
+python3 -m pip install google
+```
+
+```bash
+python3 -m pip install google.cloud.retail
+```
+
+**Tip**: Click the copy button on the side of the code box to paste the command in the Cloud Shell terminal and run it.
+
+## Set the PROJECT_NUMBER environment variable
+
+As you are going to run the code samples in your own Cloud Project, you should specify the **project_id** as an environment variable, it will be used in every request to the Retail API.
+
+You can find the ```project_number``` in the **Home/Dashboard/Project Info card**.
+
+Set the environment variable with a following command:
+```bash
+export PROJECT_NUMBER=<YOUR_PROJECT_NUMBER>
+```
+
+## Set inventory 
+
+To send the SetInventoryRequest you need to set the following fields:
+ - ```inventory``` - the inventory information to update. The allowable fields to update are: 
```

Comment:
The inventory information that you can update. The fields allowed for updating are: